.github/FUNDING.yml

```diff
@@ -0,0 +1,2 @@
+github: # TODO Replace with up to 4 GitHub Sponsors-enabled username: pyodide
```

Comment:
Should this be commented out entirely for now?

Reply:
Commented values should be fine (see e.g. https://github.com/pypy/pypy.org/blob/main/.github/FUNDING.yml), that's what they auto-suggest when creating this file from UI, and we can keep it as a todo there.